features/rake_task/rake_task.feature

```diff
@@ -36,6 +39,9 @@ Feature: Reek can be driven through its Task
         [4, 5]:Smelly#m calls @foo.bar 2 times (DuplicateMethodCall)
         [4, 5]:Smelly#m calls puts(@foo.bar) 2 times (DuplicateMethodCall)
         [3]:Smelly#m has the name 'm' (UncommunicativeMethodName)
+
+
+!!! `reek` has found smells - exiting!
```

Comment:
We [do output this on smell finds](https://github.com/troessner/reek/blob/bump_deps/lib/reek/rake/task.rb#L108) and Cucumber examples can’t indent expected output properly if there are empty lines (regardless whether they’re indented or not).

We can try either switching the `it reports` step to not be an exact match or stop displaying this line on failures; I don’t have a better solution (given that we can’t make the scenario look nice).